etcd3/client.py

```diff
@@ -922,7 +922,7 @@ def get_lease_info(self, lease_id):
         )
 
     @_handle_errors
-    def lock(self, name, ttl=60):
+    def lock(self, name, ttl=60, lock_prefix='/locks/'):
```

Comment:
refactoring `ttl` `lock_prefix` default value as `Lock` class init logic, prevent duplicate define in caller, let's set caller default value to `None`